### ISSUE TITLE
Accessibility: Android Talkback issues

### ISSUE THREAD

**Issue by joashrajin** [Automattic/pocket-casts-android#2264]
### Description

> The full player progress bar does not work with Android talkback. Swiping across the progress bar should move through the podcast but with each swipe progression leaps  back to the zero timepoint. So you can't quickly move through podcasts. Also on that page every time I touch the background I'm told  "icon view" and it gets really annoying. 
Another problem is the four buttons along the bottom of the screen which I think includes search and profile are not detected by the screen reader. So unless you have some vision you wouldn't know these buttons  were there.

### Step-by-step reproduction instructions

1. Enable Android Talkback as explained here: https://support.google.com/accessibility/android/answer/6007100?hl=en
2. Open Pocket Casts
3. Try to check if the progress bar works with talkback
4. Also check the buttons at the bottom of the page (Podcasts, Up Next, Filters, Discover, Profile)

### Did you search for existing bug reports?

- [X] I have searched for existing bug reports.

### Device, Operating system, and Pocket Casts app version

App version : 7.63 (9232)
Phone: Google - Pixel 3a - Pixel 3a - sargo
Android version: 12 SDK 32
